src/components/Advertisements/core/AdvertisementRegister/AdvertisementRegister.tsx

```
@@ -305,63 +433,58 @@
+                    />
+                  )}
+                </div>
+              )}
+              {formState.attachments.map((file, index) => (
+                <div key={index}>
+                  {file.type.startsWith('video/') ? (
+                    <video
+                      data-testid="mediaPreview"
+                      controls
+                      src={encodeURI(URL.createObjectURL(file))}
+                    />
```

Biome (1.9.4)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)